### ISSUE TITLE
Configuration > Integrations > + "Gecko" doesn't show Gecko as an option

### ISSUE THREAD

**Issue by JohnMcLear** [gazoodle/gecko-home-assistant#40]
Installed Gecko @ ``0.0.7``via HACS on 30th Jan 2022.

Nothing shows up in the UI (Configuration > Integrations > + "Gecko" doesn't show Gecko as an option)

No errors or warnings related to gecko in logs.

## System Health

version | core-2021.12.10
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.9.7
os_name | Linux
os_version | 5.10.91
arch | aarch64
timezone | Europe/London

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
Github API Calls Remaining | 4941
Installed Version | 1.19.0
Stage | running
Available Repositories | 965
Downloaded Repositories | 3

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 7.2
-- | --
update_channel | stable
supervisor_version | supervisor-2021.12.2
docker_version | 20.10.9
disk_total | 115.7 GB
disk_used | 8.0 GB
healthy | true
supported | true
board | odroid-n2
supervisor_api | ok
version_api | ok
installed_addons | File editor (5.3.3), MariaDB (2.4.0), Let's Encrypt (4.12.0), Mosquitto broker (6.0.1), Check Home Assistant configuration (3.9.0), Terminal & SSH (9.3.0), eWeLink Smart Home (1.2.3), Node-RED (10.3.4)

</details>

<details><summary>Lovelace</summary>

dashboards | 1
-- | --
resources | 0
views | 2
mode | storage

</details>

![image](https://user-images.githubusercontent.com/220864/151699136-eeae4e6e-603f-486f-8fc7-e808c5800fa4.png)

**Comment by JohnMcLear** [comment]
I discovered the installed path in my installation was
``custom_components/gecko/gecko/climate.py`` and NOT ``custom_components/gecko/climate.py``

So it's installing it into a subdir within gecko..  That doesn't seem right.

Moving everything to the root folder didn't help.

**Comment by JohnMcLear** [comment]
I did however get a follow up error

```
Exception during entry setup
17:50:33 – (ERROR) Gecko (custom integration)
Uncaught thread exception
17:50:33 – (ERROR) /usr/src/homeassistant/homeassistant/bootstrap.py
Unhandled exception in receive_handler func
17:50:03 – (ERROR) /usr/local/lib/python3.9/site-packages/geckolib/driver/udp_socket.py
```

**Comment by JohnMcLear** [comment]
My latest error is relating to a known / stale issue: https://github.com/gazoodle/gecko-home-assistant/issues/30

**Comment by gazoodle** [comment]
Should be fixed by v0.0.8
